atlasdb-console/src/test/groovy/com/palantir/atlasdb/console/groovy/TableTest.groovy

```diff
@@ -20,7 +20,8 @@ class TableTest {
     Table table
 
     final String NAME = 't'
-    final RESULT = [data: ['a', 'b']]
+    final RESULT1 = [data: ['a', 'b']]
```

Comment:
Maybe you could find more descriptive names for these. Or, even better, not give these names at all, and just say directly in the test what you want your mock to return, and what you expect back.